implants/lib/eldritch/src/file/list_impl.rs

```diff
@@ -99,20 +101,24 @@ fn create_file_from_dir_entry(dir_entry: DirEntry) -> Result<File> {
 
     let naive_datetime = match NaiveDateTime::from_timestamp_opt(timestamp, 0) {
         Some(local_naive_datetime) => local_naive_datetime,
-        None => return Err(anyhow::anyhow!("Failed to get time from timestamp for file {}", file_name)),
+        None => {
+            return Err(anyhow::anyhow!(
+                "Failed to get time from timestamp for file {}",
+                file_name
+            ))
+        }
     };
     let time_modified: DateTime<Utc> = DateTime::from_utc(naive_datetime, Utc);
-    
+
     Ok(File {
         name: file_name,
-        file_type: file_type,
+        file_type,
```

Comment:
I know why this happening but kinda gross that it’s inconsistent.

We may want to do a manual refactor to fix some of these to fall in line with our final automated formatting.

Reply:
I get this feeling. I think we can ask the linter to enforce being verbose with struct definitions

Reply:
I'd love to see what that looks like.
I see this a few other places too. Best I can figure is we do a little manual refactor with local vars using the struct param names.

Reply:
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#use_field_init_shorthand